pkg/classification/interfaces/interfaces.go

```diff
@@ -233,6 +263,19 @@ func (classifier *Classifier) FindMatchingRecipeUrl(detectionURL string) (*Recip
 
 	matchSize := 0
 	for _, recipe := range classifier.Recipes {
+		for _, recipeURL := range recipe.ExcludeURLS {
+			match, err := url.Match(detectionURL, recipeURL.RegexpMatcher)
+			if err != nil {
+				return nil, err
+			}
+
+			if match != "" {
+				return &RecipeURLMatch{
+					ExcludedURL: true,
```

Comment:
Since we have this info, we could include it (may be useful for debugging). I'll leave it up to you!

```suggestion
					ExcludedURL: true,
					DetectionURLPart: match,
					RecipeURL:        recipeURL.URL,
					RecipeUUID:       recipe.UUID,
					RecipeName:       recipe.Name,
					RecipeSubType:    recipe.SubType,
```

Reply:
Not sure yet 
I'll merge it like this let's see how useful it is :)